### PR TITLE
Add yarn setup and fix module paths

### PR DESCRIPTION
**Pull request by orlando**
**What:** Add yarn setup and fix module paths

**Why:** This should fix the dropdown component exports

**How:**

- Make lerna use independent for versioning
- Add `yarn setup` command
- fix dist paths for dc-dropdown-component package
- upgrade stencil to 2.0.3

### CODE REVIEW

File: packages/dropdown-component/package.json [hunk 1]
```diff
@@ -3,10 +3,11 @@
   "version": "1.5.0",
   "description": "Stencil dropdown component",
   "main": "dist/index.cjs.js",
-  "module": "dist/custom-elements/index.js",
-  "es2015": "dist/esm/index.mjs",
-  "es2017": "dist/esm/index.mjs",
-  "types": "dist/custom-elements/index.d.ts",
+  "module": "dist/index.js",
```
Comment (duranmla) [inline packages/dropdown-component/package.json:6]: I am not sure if this is something we should do. The version 2.0 from stencil explicitly change this check release notes at https://github.com/ionic-team/stencil/releases/tag/v2.0.0

File: lerna.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.5.0",
+  "version": "independent",
```
Comment (duranmla) [inline lerna.json:6]: This means that each component will update its version independently?

File: README.md [hunk 3]
```diff
@@ -21,8 +21,7 @@ packages/
 To get started run:
 
 ```bash
-yarn install
-lerna bootstrap
+yarn setup
```
Comment (duranmla) [inline README.md:24]: 🔥

**Review by duranmla — COMMENTED** [review]
So we are updating all components to use version 2.0.3 from stencil... how do we know that this won't bring the same issues that dropdown component currently has within the other components?

**Comment by duranmla** [comment]
I'm hesitating to merge this because the `dist` paths from dropdown component were generated automatically and we are changing them manually. What is good tho, is that we are unifying everything to 2.0.3.

If we don't have a way to test this before merge then let's try this out.
🏹

File: packages/dropdown-component/package.json (hunk 1 above)
Comment (orlando) [reply packages/dropdown-component/package.json:6]: yeah we are using that, you can compare the values

File: lerna.json (hunk 2 above)
Comment (orlando) [reply lerna.json:6]: correct

**Comment by orlando** [comment]
ok I'll :shipit:

**Pull request merged**